src/StreamService.Data.Provider/IDataProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StreamService.Data.Provider
+{
+    public interface IDataProvider

```

Comment:
IDataProvider inherits IBaseDataProvider, don't forget to add IBaseDataProvider methods to StreamServiceDbContext

Reply:
IDataProvider was deleted